tests/parametric/conftest.py

```diff
@@ -191,36 +191,39 @@ def golang_library_factory():
 
 
 def dotnet_library_factory():
-    dotnet_appdir = os.path.join("utils", "build", "docker", "dotnet", "parametric")
+    dotnet_appdir = os.path.join("utils", "build", "docker", "dotnet_http", "parametric")
     dotnet_absolute_appdir = os.path.join(_get_base_directory(), dotnet_appdir)
     dotnet_reldir = dotnet_appdir.replace("\\", "/")
     server = APMLibraryTestServer(
         lang="dotnet",
-        protocol="grpc",
-        container_name="dotnet-test-client",
-        container_tag="dotnet7_0-test-client",
+        protocol="http",
+        container_name="dotnet-test-api",
+        container_tag="dotnet8_0-test-api",
         container_img=f"""
-FROM mcr.microsoft.com/dotnet/sdk:7.0
-RUN apt-get update && apt-get install dos2unix
-WORKDIR /app
+FROM mcr.microsoft.com/dotnet/sdk:8.0
 
-# Opt-out of .NET SDK CLI telemetry (prevent unexpected http client spans)
-ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
+RUN apt-get update && apt-get install -y dos2unix --no-install-recommends
+RUN apt-get update && apt-get install -y curl
+USER root
+WORKDIR /app
 
 # ensure that the Datadog.Trace.dlls are installed from /binaries
-COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
+COPY install_ddtrace.sh query-versions.fsx binaries* /binaries/
 RUN dos2unix /binaries/install_ddtrace.sh
 RUN /binaries/install_ddtrace.sh
 
 # restore nuget packages
-COPY ["{dotnet_reldir}/ApmTestClient.csproj", "{dotnet_reldir}/nuget.config", "{dotnet_reldir}/*.nupkg", "./"]
-RUN dotnet restore "./ApmTestClient.csproj"
+COPY ["ApmTestApi.csproj", "nuget.config", "*.nupkg", "./"]
+RUN dotnet restore "./././ApmTestApi.csproj"
```

Comment:
Another small nit, even though it shouldn't change the behavior:

```suggestion
RUN dotnet restore "./ApmTestApi.csproj"
```